diesel/src/r2d2.rs

```diff
@@ -8,8 +8,8 @@ pub use self::r2d2::*;
 
 /// A re-export of [`r2d2::Error`], which is only used by methods on [`r2d2::Pool`].
 ///
-/// [`r2d2::Error`]: ../../r2d2/struct.Error.html
-/// [`r2d2::Pool`]: ../../r2d2/struct.Pool.html
+/// [`r2d2::Error`]: crate::r2d2::Error
+/// [`r2d2::Pool`]: crate::r2d2::Pool
```

Comment:
These imports are wrong, they should refer to the `r2d2` crate not the reexport modul in diesel itself:
```suggestion
/// [`r2d2::Error`]: r2d2::Error
/// [`r2d2::Pool`]: r2d2::Pool
```